pkg/asset/agent/image/agentimage.go

```diff
@@ -181,7 +182,10 @@ func (a *AgentImage) appendAgentFilesToInitrd(additionalFiles []string) error {
 	// Add a dracut hook to copy the files. The $NEWROOT environment variable is exported by
 	// dracut during the startup and it refers the mountpoint for the root filesystem.
 	dracutHookScript := `#!/bin/sh
-cp -R /agent-files/* $NEWROOT/usr/local/bin/`
+cp -R /agent-files/* $NEWROOT/usr/local/bin/
```

Comment:
In general I agree, in the sense that this is just a first rough mechanism to copy over stuff from the initramfs to the disk to have an initial integration, and it could be definitely improved as our requirements increase. We could follouwup in a subsequent pr